### PR TITLE
Add ODFE 1.11.0 release notes

### PR DESCRIPTION
**Pull request by camerski**
By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.

### CODE REVIEW

File: release-notes/opendistro-for-elasticsearch-release-notes-1.11.0.md [hunk 1]
```diff
@@ -0,0 +1,315 @@
+# Open Distro for Elasticsearch 1.11.0 Release Notes
+
+## Release Highlights
+
+## Release Details
+
+You can also track upcoming features in Open Distro for Elasticsearch by watching the code repositories or checking the project website (https://opendistro.github.io/for-elasticsearch/features/comingsoon.html).
```
Comment (gaiksaya) [inline release-notes/opendistro-for-elasticsearch-release-notes-1.11.0.md:7]: Square brackets around project website [project website]  (https://opendistro.github.io/for-elasticsearch/features/comingsoon.html)

File: release-notes/opendistro-for-elasticsearch-release-notes-1.11.0.md [hunk 2]
```diff
@@ -0,0 +1,315 @@
...
+* Add paragraph components and UI helpers ([#28](https://github.com/opendistro-for-elasticsearch/kibana-notebooks/pull/28))
+* Add UI components for App, main, notebooks and note_buttons ([#27](https://github.com/opendistro-for-elasticsearch/kibana-notebooks/pull/27))
+* Add new route for visualizations ([#25](https://github.com/opendistro-for-elasticsearch/kibana-notebooks/pull/25))
+* Add API Routes and Zeppelin Backend Adaptor ([#24](https://github.com/opendistro-for-elasticsearch/kibana-notebooks/pull/24))
+* Add .DS_Store in gitignore, added header in app.tsx ([#23](https://github.com/opendistro-for-elasticsearch/kibana-notebooks/pull/23))
+* Initial Commit - new platform, gitignore, license header ([#18](https://github.com/opendistro-for-elasticsearch/kibana-notebooks/pull/18))
+* Build sample plugin RPM ([#3](https://github.com/opendistro-for-elasticsearch/kibana-notebooks/pull/3))
+
+
+### Knn
```
Comment (gaiksaya) [inline release-notes/opendistro-for-elasticsearch-release-notes-1.11.0.md:34]: K-NN

File: release-notes/opendistro-for-elasticsearch-release-notes-1.11.0.md [hunk 3]
```diff
@@ -0,0 +1,315 @@
...
+
+### Performance Analyzer
+* Batch Metrics API ([#159](https://github.com/opendistro-for-elasticsearch/performance-analyzer/pull/159))
+
+
+### Perftop
+* Feature temperature rca ([#52](https://github.com/opendistro-for-elasticsearch/perftop/pull/52))
+
+
+### Sql
```
Comment (gaiksaya) [inline release-notes/opendistro-for-elasticsearch-release-notes-1.11.0.md:46]: SQL all caps

File: release-notes/opendistro-for-elasticsearch-release-notes-1.11.0.md (hunk 1 above)
Comment (camerski) [reply release-notes/opendistro-for-elasticsearch-release-notes-1.11.0.md]: Updated

File: release-notes/opendistro-for-elasticsearch-release-notes-1.11.0.md (hunk 2 above)
Comment (camerski) [reply release-notes/opendistro-for-elasticsearch-release-notes-1.11.0.md]: Updated

File: release-notes/opendistro-for-elasticsearch-release-notes-1.11.0.md (hunk 3 above)
Comment (camerski) [reply release-notes/opendistro-for-elasticsearch-release-notes-1.11.0.md]: Updated

**Review by MMakaria — COMMENTED** [review]
Looks good ! thanks

**Pull request merged**